src/Game.cxx

```diff
@@ -269,8 +268,8 @@ void Game::run(bool SkipMenu)
   else
   {
 	//playing song in stereo with left and right locations
-	m_GameClock.createRepeatedTask(8min, [this]() { m_AudioMixer.play(AudioTrigger::MainTheme,Coordinate3D{2, 0, 1}); });
-	m_GameClock.createRepeatedTask(8min, [this]() { m_AudioMixer.play(AudioTrigger::MainTheme,Coordinate3D{-2, 0, 1}); });
+	
+	m_GameClock.createRepeatedTask(8min, [this]() { m_AudioMixer.play(AudioTrigger::MainMenu,Coordinate3D{0,0.5,0.1}); });
```

Comment:
Why is this `MainMenu` now?

Reply:
Sorry. This was a lazy copy paste mistake. I corrected it to use a main theme now.